src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerLogStepExecution.java

```diff
@@ -33,9 +35,22 @@
     private static final long serialVersionUID = 5588861066775717487L;
     private static final transient Logger LOGGER = Logger.getLogger(ContainerLogStepExecution.class.getName());
 
-    private final ContainerLogStep step;
+    @SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "not needed on deserialization")
+    private transient final ContainerLogStep step;
     private transient KubernetesClient client;
 
+    @Override
+    // TODO Revisit for JENKINS-40161
+    public void onResume() {
```

Comment:
I'm not convinced this is the proper implementation for this step. The parent class `SynchronousNonBlockingStepExecution` makes it clear that resuming such a step is unsupported, the bulk of the execution being run in a different thread, which is not resumed on restart.

As I understand it, the current `onResume` is just a no-op, so you may as well remove injection of 'client' as it won't be used in later calls.

Reply:
@jglick you didn't raise this point before, do you agree with @Vlatombe  ?